allure-results/environment.xml

```diff
@@ -0,0 +1,14 @@
+<environment>
```

Comment:
It will be hard to maintain this information manually, find the way to add it programmatically